### PR TITLE
[fix bug 1471034] Add S03E01

### PR DESCRIPTION
**Pull request by craigcook**
Let's see if I did this right...

### CODE REVIEW

File: content/episodes/2018-07-02-S03E01-the-grand-bargain.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
+---
+categories: "episodes"
+date: 2018-07-02T09:00:00-07:00
+description: "We’re told from a young age to “accept the things we cannot change.” But should this be the case online as well? We click “Accept” every day, but often don’t know what we’re giving away. Is it a fair trade, and should we demand a better bargain? [Veronica Belmont](https://twitter.com/Veronica) and special guest [Dave Pell](https://twitter.com/davepell) explore if what we get for what we give online is a good deal. We hear how one man’s HIV status was exposed without permission, how a massive data-mining company is using our information to predict how we'll behave, and why on earth our email inboxes are filling up with privacy policies."
+embed: "7a906834"
+mp3: "https://dts.podtrac.com/redirect.mp3/audio.simplecast.com/7a906834"
+number: 16
```
Comment (craigcook) [inline content/episodes/2018-07-02-S03E01-the-grand-bargain.md]: I wasn't sure of the episode number... is this 16 or 17 in the total? Or 19? Do we count the 0 episodes but skip the bonus episodes?

File: content/episodes/2018-07-02-S03E01-the-grand-bargain.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+    Nora Young discusses the GDPR in this episode. Here are [13 more things you need to know about the GDPR](https://blog.mozilla.org/internetcitizen/2018/05/23/gdpr-mozilla/).
+
+    Beyond GDPR, [check out what else is changing your online rights](http://blog.mozilla.org/internetcitizen/2018/06/25/changes-to-online-rights/).
+
+    The rest of Jaron Lanier's talk can be heard on [TED Talks Daily](https://player.fm/series/tedtalks-audio/how-we-need-to-remake-the-internet-jaron-lanier).
+
+    Find Dave Pell's NextDraft newsletter [here](https://nextdraft.com/).
+
+    And, [click here](https://www.mozilla.org/en-US/internet-health/privacy-security/) for Mozilla's take on privacy and the trade-offs we make online.
```
Comment (jpetto) [inline content/episodes/2018-07-02-S03E01-the-grand-bargain.md]: Nit - remove `en-US` from the URL.

File: content/episodes/2018-07-02-S03E01-the-grand-bargain.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+description: "We’re told from a young age to “accept the things we cannot change.” But should this be the case online as well? We click “Accept” every day, but often don’t know what we’re giving away. Is it a fair trade, and should we demand a better bargain? [Veronica Belmont](https://twitter.com/Veronica) and special guest [Dave Pell](https://twitter.com/davepell) explore if what we get for what we give online is a good deal. We hear how one man’s HIV status was exposed without permission, how a massive data-mining company is using our information to predict how we'll behave, and why on earth our email inboxes are filling up with privacy policies."
+embed: "7a906834"
+mp3: "https://dts.podtrac.com/redirect.mp3/audio.simplecast.com/7a906834"
+number: 16
+ogdescription: "Explore the tradeoffs you make online everyday, and learn why on earth your email inbox is filling up with privacy policies."
+ogtitle: "IRL Podcast: The Grand Bargain"
+season: 3
+seasonepisode: 1
+shownotes: |
+    Tom Hayes works for an organization called Beyond Positive. [Learn more](http://www.beyondpositive.org/).
```
Comment (jpetto) [inline content/episodes/2018-07-02-S03E01-the-grand-bargain.md]: Nit - just link the "Beyond Positive" text. (We have more say/control on copy here than on moz.org.)

File: content/episodes/2018-07-02-S03E01-the-grand-bargain.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+
+    Nora Young discusses the GDPR in this episode. Here are [13 more things you need to know about the GDPR](https://blog.mozilla.org/internetcitizen/2018/05/23/gdpr-mozilla/).
+
+    Beyond GDPR, [check out what else is changing your online rights](http://blog.mozilla.org/internetcitizen/2018/06/25/changes-to-online-rights/).
+
+    The rest of Jaron Lanier's talk can be heard on [TED Talks Daily](https://player.fm/series/tedtalks-audio/how-we-need-to-remake-the-internet-jaron-lanier).
+
+    Find Dave Pell's NextDraft newsletter [here](https://nextdraft.com/).
```
Comment (jpetto) [inline content/episodes/2018-07-02-S03E01-the-grand-bargain.md]: If you feel like making this language more natural (e.g. dropping the "here" bit), feel free. 😀

File: content/episodes/2018-07-02-S03E01-the-grand-bargain.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+
+    And, [click here](https://www.mozilla.org/en-US/internet-health/privacy-security/) for Mozilla's take on privacy and the trade-offs we make online.
+url: /season3/episode1/
+aliases:
+    - /episode15/
```
Comment (jpetto) [inline content/episodes/2018-07-02-S03E01-the-grand-bargain.md]: I don't _think_ we need these aliases going forward. AFAIK, they are only for legacy episodes published under the old URL scheme.

File: content/episodes/2018-07-02-S03E01-the-grand-bargain.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
+---
+categories: "episodes"
+date: 2018-07-02T09:00:00-07:00
+description: "We’re told from a young age to “accept the things we cannot change.” But should this be the case online as well? We click “Accept” every day, but often don’t know what we’re giving away. Is it a fair trade, and should we demand a better bargain? [Veronica Belmont](https://twitter.com/Veronica) and special guest [Dave Pell](https://twitter.com/davepell) explore if what we get for what we give online is a good deal. We hear how one man’s HIV status was exposed without permission, how a massive data-mining company is using our information to predict how we'll behave, and why on earth our email inboxes are filling up with privacy policies."
+embed: "7a906834"
+mp3: "https://dts.podtrac.com/redirect.mp3/audio.simplecast.com/7a906834"
```
Comment (jpetto) [inline content/episodes/2018-07-02-S03E01-the-grand-bargain.md]: This doesn't actually link to an mp3 file. 😞 We need a different link.

File: content/episodes/2018-07-02-S03E01-the-grand-bargain.md (hunk 1 above)
Comment (jpetto) [reply content/episodes/2018-07-02-S03E01-the-grand-bargain.md]: This value is actually used in the Apple and RadioPublic subscribe links. Maybe we should talk with Pacific Content about updating what's used there so we can drop this legacy value. (Guessing we can switch to something including season + episode.)

File: content/episodes/2018-07-02-S03E01-the-grand-bargain.md (hunk 6 above)
Comment (craigcook) [reply content/episodes/2018-07-02-S03E01-the-grand-bargain.md]: Looks like it needs `.mp3` at the end. That was missing in the spreadsheet.

**Comment by craigcook** [comment]
Updated!

**Comment by jpetto** [comment]
Would be good to get this on dev for the IRL team to see. (See the README.)

**Pull request merged**